components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java

```diff
@@ -49,195 +49,197 @@
-								break;
-							case OBJECT:
-								object[j] = eventStreamAttributeArray[j];
-								break;
-						}
-					}
-					//send events
-					for (String key : inputHandlerMap.keySet()) {
-						if (key.equals(inputStreamId)) {
-							inputHandlerMap.get(key).send((startSetTime +
-							                               (System.currentTimeMillis() -
-							                                startSystemTime)), object);
-						}
-					}
-				} else if (delayPatternMatcher.find()) {
-					Thread.sleep(Long.parseLong(delayPatternMatcher.group(2)));
-				} else {
-					if (!inputStreamEventArray[i].equals("")) {
-						errMsg = "Error in event \"  " +
-						         inputStreamEventArray[i] +
-						         "\n\"." +
-						         " Expected format: &lt;eventStreamName&gt;=[&lt;attribute1&gt;,&lt;attribute2&gt;]";
-						throw new Exception(errMsg);
-					}
-				}
-			}
-			Thread.sleep(500);
-			return map;
-		} catch (Throwable e) {
-			//catch throwable since there could be execution plan validation exceptions
-			errMsg = "Error occurred while processing. " + e.getMessage();
-			log.error(errMsg, e);
-			throw new Exception(errMsg, e);
-		} finally {
-			if (!executionPlanRuntime.equals(null)) {
-				executionPlanRuntime.shutdown();
-			}
-		}
-	}
-
-	/**
-	 * Create time stamp for the given date and time
-	 *
-	 * @param dateTime date and time to begin the process
-	 */
-	private long createTimeStamp(String dateTime) throws Exception {
-		SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
-		Date date;
-		try {
-			date = dateFormatter.parse(dateTime);
-		} catch (Exception e) {
-			errMsg = "Error occurred while parsing date " + e.getMessage();
-			log.error(errMsg, e);
-			throw new Exception(errMsg, e);
-		}
-		long timeStamp = date.getTime();
-		return timeStamp;
-	}
-
-	/**
-	 * Load data sources configuration
-	 *
-	 * @param siddhiManager SiddhiManager object
-	 */
-	public static void loadDataSourceConfiguration(SiddhiManager siddhiManager){
-		try {
-			int tenantId = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantId();
-			if (tenantId > -1) {
-				DataSourceManager.getInstance().initTenant(tenantId);
-			}
-			List<CarbonDataSource> dataSources = SiddhiTryItValueHolder.getDataSourceService().getAllDataSources();
-			for (CarbonDataSource cds : dataSources) {
-				try {
-					if (cds.getDSObject() instanceof DataSource) {
-						siddhiManager.setDataSource(cds.getDSMInfo().getName(), (DataSource) cds.getDSObject());
-					}
-				} catch (Exception e) {
-					log.error("Unable to add the datasource" + cds.getDSMInfo().getName(), e);
-				}
-			}
-		} catch (DataSourceException e) {
-			log.error("Unable to populate the data sources in Siddhi engine.", e);
-		}
-	}
+    private static Log log = LogFactory.getLog(SiddhiTryItClient.class);
+    private static Gson gson = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
+    private String errMsg = "";
```

Comment:
Sure, I have fixed it in PR:https://github.com/wso2/carbon-event-processing/pull/179